src/main/java/Product.java

```diff
@@ -0,0 +1,16 @@
+public class Product extends Record{
+
+    private String category;
+
+    public Product (String name, float price, String date, String productType){
+        super( name, price, date);
+        this.category = productType;
```

Comment:
category and product type are different. 
category: product/ subscription
product type is only for product class: fashion, food, electronics

please make the modifications accordingly! it is quite impt